packages/server-boilerplate/package.json

```diff
@@ -26,7 +26,7 @@
   "dependencies": {
     "@babel/polyfill": "^7.0.0",
     "@tupaia/access-policy": "3.0.0",
-    "@tupaia/api-client": "2.0.1",
+    "@tupaia/api-client": "^2.0.1",
```

Comment:
Ah sorry @IgorNadj can we leave this as exact matching? It means it gets it from the relative path in the mono-repo rather than npm, which is really handy as well as being a bit more efficient.

Reply:
Ah sorry that was a mistake! I have to fix something else, will revert it in the same change.